### ISSUE TITLE
Collaborators' Headshots

### ISSUE THREAD

**Issue by vmb4ah** [vmb4ah/TBTA-Collaborators-Photos#1]
<img width="289" alt="vanessa braganza" src="https://user-images.githubusercontent.com/19616403/39971623-5f466a7e-56cc-11e8-9a8a-7499da892872.png">
<img width="291" alt="rachel newman" src="https://user-images.githubusercontent.com/19616403/39971625-68397ea0-56cc-11e8-93af-fe4eb3809d98.png">
<img width="205" alt="melanie pace" src="https://user-images.githubusercontent.com/19616403/39971636-82b01f00-56cc-11e8-8c91-c9ddb8f29520.png">
<img width="263" alt="tierney vial" src="https://user-images.githubusercontent.com/19616403/39971630-721045d0-56cc-11e8-82cf-055e42505a88.png">
<img width="459" alt="katherine donally" src="https://user-images.githubusercontent.com/19616403/39971643-8915d7fe-56cc-11e8-9097-ebbec1ef6ee7.png">

**Comment by jeremyboggs** [comment]
do you have higher resolution JPEGs for these? Worry that, in uploading them to this ticket, that Github reduce their filesize and resolution.

**Comment by vmb4ah** [comment]
No, these are the only ones I have. Should I email the lady who sent them
and ask for higher resolution? I'm not sure whether she'll be able to
procure them.

**Comment by jeremyboggs** [comment]
Ok, will make these work.
